align_system/algorithms/hybrid_kaleido_adm.py

```diff
@@ -30,4 +30,5 @@ def choose_action(self, scenario_state, available_actions, alignment_target, **k
             action_to_take,
             dialog)
 
-        return action_to_take
+        choice_info = {}
```

Comment:
Nit: Not sure this needs to be changed now (and David would probably know better), but I'm guessing at a minimum the hybrid Kaleido ADM will pass along the dictionary from the non-hybrid Kaleido ADM

Reply:
Ah good catch, fixed here https://github.com/ITM-Kitware/align-system/pull/122/commits/b38337170be9517731b7197833ce3f2e7efed6ec